shaky/build.gradle

```diff
@@ -30,15 +30,15 @@ android {
 }
 
 dependencies {
-    compile 'com.squareup:seismic:1.0.2'
-    compile "com.android.support:appcompat-v7:$supportLibrariesVersion"
-    compile "com.android.support:recyclerview-v7:$supportLibrariesVersion"
-    compile "com.android.support:support-v4:$supportLibrariesVersion"
-    compile "com.android.support:support-annotations:$supportLibrariesVersion"
-
-    testCompile 'junit:junit:4.12'
-    testCompile 'org.mockito:mockito-core:1.9.5'
-    testCompile 'org.robolectric:robolectric:3.0'
+    api 'com.squareup:seismic:1.0.2'
+    implementation "androidx.appcompat:appcompat:$supportLibrariesVersion"
+    implementation "androidx.recyclerview:recyclerview:$supportLibrariesVersion"
+    implementation "androidx.legacy:legacy-support-v4:$supportLibrariesVersion"
+    implementation "androidx.annotation:annotation:$supportLibrariesVersion"
```

Comment:
Just curious, does google also ensure the different major version compatibility? We can already use support library (not X) if the major version is the same. 

I wonder this $supportLibrariesVersion is just trying to make sure it's more likely to be stable using different support libraries.

Reply:
I believe it was an actual warning Android Studio would throw at you if they weren't the exact same version. https://stackoverflow.com/questions/42374151/all-com-android-support-libraries-must-use-the-exact-same-version-specification

Either way, this won't work anymore with AndroidX since they will be updated individually